src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -68,11 +68,12 @@
  ::catalogue-table-rows
  (fn [_ _]
    [(rf/subscribe [::catalogue])
-    (rf/subscribe [:language])])
- (fn [[catalogue language] _]
+    (rf/subscribe [:language])
+    (rf/subscribe [:default-language])])
```

Comment:
The `:language` subscription already falls back to `:default-language`, so the fallback doesn't need to be done again.